src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -197,11 +199,49 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     mpu: props =>
       isAdsEnabled ? <MpuContainer {...props} slotType="mpu" /> : null,
     wsoj: props => (
-      <CpsRecommendations
-        {...props}
-        parentColumns={gridColsMain}
-        items={recommendationsData}
-      />
+      // 004_brasil_recommendations_experiment
+      <OptimizelyExperiment experiment={OPTIMIZELY_CONFIG.experimentId}>
+        {variation => {
+          if (variation === 'control' || !variation) {
+            return (
+              <CpsRecommendations
+                {...props}
+                parentColumns={gridColsMain}
+                items={recommendationsData}
+              />
+            );
+          }
+          if (variation === 'variation_1') {
+            const unirecsContentRecommendationData = path(
+              ['datalabContentRecommendations'],
+              pageData,
+            );
+
+            return (
+              <CpsRecommendations
+                {...props}
+                parentColumns={gridColsMain}
+                items={unirecsContentRecommendationData}
+              />
+            );
+          }
+          if (variation === 'variation_3') {
```

Comment:
Just thinking should this be `variation_2`? Think its called `variation_3` in Optimizely as well, so may need changed.

Jem mentioned that we could name these variations differently, relating more to what they are doing, e.g. `hybrid_recs` etc instead of just `variation_x`

Reply:
this would be cleaner and easier to follow 👍

Reply:
Done 👍 . Changed to `hybrid_recs` and `content_recs`.